src/utils/HSwitchGlobalVars.res

```diff
@@ -1,5 +1,7 @@
 @val external appVersion: string = "appVersion"
 
+let hyperSwitchAppVersion = Window.env.versionNumber->Option.getOr("2024.04.10.0")
```

Comment:
Instead of random date we can add today's date and show accordingly in default case